herbert_core/classes/MPIFramework/parallel_call/parallel_call.m

```diff
@@ -1,23 +1,21 @@
 function varargout = parallel_call(func, args, varargin)
 
-    [nWorkers] = get(parallel_config, 'parallel_workers_number');
+    nWorkers = get(parallel_config, 'parallel_workers_number');
 
     func_info = functions(func);
 
     switch func_info.function
       case 'sqw_eval'
 
-        w = args{1};
-        args = args(2:end);
-
-        varargout{1} = parallel_sqw_eval(w, args);
+        varargout{1} = parallel_sqw_eval(func, nWorkers, args);
 
       case 'multifit_func_eval'
 
-        w = args{1};
-        args = args(2:end);
+        % mf_func_eval signature:
+        %multifit_func_eval (wmask, xye, fun_wrap, bfun_wrap, pin_wrap, bpin_wrap,...
+        %    f_pass_caller, bf_pass_caller, pfin, p_info, output_type)
```

Comment:
do we want to keep this?

Reply:
I personally think it's useful for determining why it's structured the way it is. I would probably prefer putting the other function signature at top of the other case.